ClientApp/viking-vault/src/components/AccountPage.tsx

```diff
@@ -40,6 +66,7 @@ class AccountPage extends React.Component<any, IAccountState>{
         }
         else
         {
+            setInterval(this.updateExchangingRates, 5000);
```

Comment:
You should extract this timeout period either in a constant in this file, or store it in the constants file, then when you want to change the timeout interval, you simply go there and change it rather than hunt it down to see where it's used